Makefile

```diff
@@ -35,3 +52,53 @@ unit-test:
 
 build: $(SOURCES)
 	go build $(GOBUILDFLAGS) -ldflags="$(GOLDFLAGS)" -o ./update-service-operator ./
+
+# find or download controller-gen
+# download controller-gen if necessary
+controller-gen:
+ifeq (, $(shell which controller-gen))
+        @{ \
```

Comment:
[images][1]:

```
Makefile:60: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati-operator/93/pull-ci-openshift-cincinnati-operator-master-images/1334930754748026880#1:build-log.txt%3A71

Reply:
I did. Bit by cut-and-paste once again :) Fixed in [27ac93d](https://github.com/openshift/cincinnati-operator/pull/93/commits/27ac93d60d45be1194a27a32f9e525801d2456d3)